docs/developer/developer-manual.html

```diff
@@ -438,6 +440,28 @@ <h3 id="ci-azure-pipelines">Azure Pipelines</h3>
   <li>Click "run".</li>
 </ul>
 
+<h4 id="ci-increasing-quota">Increasing quota</h4>
+
+<p>
+  It may be the case that you obtain only 1800 minutes per month, and no parallelism, and only for private projects.
+  In this case, you need to ask Azure to lift all three restrictions.  Send mail such as the following:
```

Comment:
```suggestion
  In this case, you need to ask Azure to lift all three restrictions. Azure DevOps provides a <a href="https://aka.ms/azpipelines-parallelism-request">Microsoft Forms</a> through which you can request increased public project concurrency for validated open-source projects. You may also send mail such as the following:
```

Reply:
Did you get the desired parallelism from this form?

For me, I filled out the form and they accepted it, but only granted the parallelism under the mentioned three limitations (private, 1800 mins, 1 job at a time). Because by default you have no parallelism at all. I'd say the process is to
1. fill out the form
2. if the granted parallelism has those restrictions, send the email as described

Can you confirm that?

Reply:
They haven't given me a response to my form yet, but I expect it will have the same outcome as yours. Your suggested process sounds like the best way to go about it.

Reply:
For clarity, does that mean that my original wording is preferred?

Reply:
If I'm not mistaken, the form is not mentioned at all currently. I think the text should say:
1. By default, you have no parallelism.
2. To request parallelism, fill out this form.
3. If the granted parallelism is limited by these 3 constraints, send the email as described.

I'm not 100% certain this is the exact process I went through, it's been some time, so let me know if you agree or disagree with this.

Reply:
Thanks for the clarification.  @skehrli could you re-review?

Reply:
Looks good to me now